### PR TITLE
Investigate travis failures

### PR DESCRIPTION
**Pull request by aparthibanpaypal**

### CODE REVIEW

**Review by SaiNarendran — DISMISSED** [review]

File: Demo/AppDelegate.swift [hunk 1]
```diff
@@ -32,6 +32,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.tintColor = .systemBlue
         window?.backgroundColor = Theme.UITableViewController.backgroundColor
 //        ThemeManager.applyWhiteTheme()
+
```
Comment (jsaini-hw) [inline Demo/AppDelegate.swift:35]: Why this change?

**Review by jsaini-hw — APPROVED** [review]

File: Demo/AppDelegate.swift (hunk 1 above)
Comment (jsaini-hw) [reply Demo/AppDelegate.swift:35]: To start travis CI build again you don't need to commit anything, just do a push from your branch and it will trigger travis ci build

**Review by SaiNarendran — APPROVED** [review]

**Review by rbanikPP — APPROVED** [review]

**Comment by jsaini-hw** [comment]
changes merged as part of another PR
